Pipeline/routes/route.py

```diff
@@ -109,9 +109,23 @@ def get_graphs_per_module():
         abort(400, "Failed to retrieve representative graphs. Please check your inputs. Error: " + str(e))
 
 
-@routes.route('/rnamigos', methods=['GET'])
-def greeting_rna_migos():
-    return RnaMigos.greeting()
+@routes.route('/rnamigos_string', methods=['POST'])
```

Comment:
the url path shouldn't have an underscore. If you look at the convention [here](https://restfulapi.net/resource-naming/), you should have `/rnamigos-string`

![image](https://user-images.githubusercontent.com/35381483/111016507-69d9d900-837c-11eb-967d-5d808fa584bd.png)

Reply:
Makes sense for the endpoint name, but I believe it's better 'style' for python to have method names use _ instead of -. Therefore, I'll change the endpoint to using - as suggested, but I'll keep the _ for the method names. Thanks!